curation_concerns-models/app/services/curation_concerns/derivative_path.rb

```diff
@@ -7,8 +7,19 @@ def derivative_path_for_reference(object, destination_name)
         derivative_path(object, extension_for(destination_name), destination_name)
       end
 
+      # Returns all derivatives on the filesystem for this object.
+      def derivatives_for_reference(object)
+        Dir.glob(root_path(object).join("*")).select do |path|
+          path.start_with?(derivative_path(object, "", ""))
```

Comment:
What do you think about having `""` be the default value for those two args in `def derivative_path` ?